INSTALL.md

```diff
@@ -4,25 +4,40 @@
 ### 1. Installing EtherLab
 The proposed development builds upon the [IgH EtherCAT Master](https://etherlab.org/en/ethercat/). Installation steps are summarized here:
 ```shell
+$ sudo apt-get update
+$ sudo apt-get upgrade
+$ sudo apt install git
 $ git clone https://gitlab.com/etherlab.org/ethercat.git
 $ cd ethercat
 $ git checkout stable-1.5
-```
-```shell
-$ ./bootstrap # to create the configure script, if downloaded from the repository
-
-$ ./configure --prefix=/usr/local/etherlab  --disable-8139too --enable-generic # Ethernet driver e1000e not supported for kernels 4.X
+$ sudo apt-get install autoconf
+$ sudo apt-get install libtool
+$ sudo apt-get install pkg-config
+$ sudo apt-get install make
+$ sudo apt-get install build-essential
+$ sudo apt-get install net-tools
```

Comment:
maybe on a single line for better readability ?

Reply:
okay. corrected this in the latest commit.